src/Stratis.Bitcoin/Utilities/NodeStats.cs

```diff
@@ -19,6 +19,8 @@ public interface INodeStats
 
         /// <summary>Collects benchmark stats.</summary>
         string GetBenchmark();
+
+        bool NodeIsInIbd { get; set; }
```

Comment:
This shouldn't be on the interface. 

Stats are always for display so just having this info returned by `string GetStats()` is enough

Reply:
Just inject `InitialBlockDownloadState` in to stats and get that value when log is called